### ISSUE TITLE
Sampler hitting NaN when running with free neutrino mass and H0

### ISSUE THREAD

**Issue by Chris-Pedersen** [andreufont/LyaCosmoParams#95]
After ~3k steps, I hit:

```
  File "cmb_free_nu_h_3igm.py", line 147, in <module>
    sampler.run_sampler(args.burn_in,args.nsteps,log_prob,parallel=args.parallel)
  File "/home/chrisp/.local/lib/python3.6/site-packages/emcee_sampler.py", line 181, in run_sampler
    progress=self.progress):
  File "/home/chrisp/.local/lib/python3.6/site-packages/emcee/ensemble.py", line 343, in sample
    state, accepted = move.propose(model, state)
  File "/home/chrisp/.local/lib/python3.6/site-packages/emcee/moves/red_blue.py", line 93, in propose
    new_log_probs, new_blobs = model.compute_log_prob_fn(q)
  File "/home/chrisp/.local/lib/python3.6/site-packages/emcee/ensemble.py", line 456, in compute_log_prob
    raise ValueError("Probability function returned NaN")
ValueError: Probability function returned NaN
```

This might be tricky to reproduce as the other sampler run with free H0 and neutrino mass has been going for >24h without any problems. But I will load up this likelihood object in a notebook and see if we hit any NaNs on the edge of the prior volume, for example.

It would also be neat to make it so if this happens again, the error will print the point in likelihood space that returned a NaN

**Comment by andreufont** [comment]
Agreed. Note that we might be pushing the limits of the setup here, with very poorly constrained parameters that might take us to dangerous places. You could also add a debug mode the prints the chain as we go, or prints a line with parameter calues at each likelihood call.

**Comment by Chris-Pedersen** [comment]
Fixed in #110
